### PR TITLE
Add default values to --help parameters

### PR DESCRIPTION
**Pull request by PalumboN**
I checked with the values from my running image, I guess that there are the default values, but better if somebody can re-check them.

On other side, I don't like the parameter `avoidSearchingSegmentsWithPinnedObjects` because it talk in negated way. For me it should be called `searchSegmentsWithPinnedObjects (default: true)`.

### CODE REVIEW

**Review by guillep — APPROVED** [review]
Looks ok to me.

I agree with the double negation.
The explanation of the parameter could be improved a bit too :)

File: src/parameters/parameters.c [hunk 1]
```diff
@@ -438,19 +438,19 @@ vm_printUsageTo(FILE *out)
...
+"  --minPermSpaceSize=<size>[mk]        Sets the min size of the permanent space (default: 0k)\n"
 "                                       It is possible to use k(kB), M(MB) and G(GB).\n"
-"  --stackPageSize=<size>[mk]           Sets the size of each stack page\n"
+"  --stackPageSize=<size>[mk]           Sets the size of each stack page (default: 8k)\n"
 "                                       It is possible to use k(kB), M(MB) and G(GB).\n"
-"  --workingDirectory=<dir>				It sets the working directory for the running image.\n"
+"  --workingDirectory=<dir>		It sets the working directory for the running image.\n"
 "  --avoidSearchingSegmentsWithPinnedObjects\n"
-"                                       When pinning young objects, the objects are clonned into the old space.\n"
+"                                       When pinning young objects, the objects are clonned into the old space. (default: false)\n"
```
Comment (astares) [inline src/parameters/parameters.c]: typo:  "cloned" instead of "clonned"

Comment (PalumboN) [reply src/parameters/parameters.c]:
```suggestion
"                                       When pinning young objects, the objects are cloned into the old space. (default: false)\n"
```

Comment (astares) [reply src/parameters/parameters.c]: Thanks Nahuel!

**Comment by guillep** [comment]
Let's go with it and add an issue for the follow up things.

Thank you very much Mr Nahuel

File: src/parameters/parameters.c [hunk 2]
```diff
@@ -438,19 +438,19 @@ vm_printUsageTo(FILE *out)
...
+"  --stackPageSize=<size>[mk]           Sets the size of each stack page (default: 8k)\n"
 "                                       It is possible to use k(kB), M(MB) and G(GB).\n"
-"  --workingDirectory=<dir>				It sets the working directory for the running image.\n"
+"  --workingDirectory=<dir>		It sets the working directory for the running image.\n"
 "  --avoidSearchingSegmentsWithPinnedObjects\n"
-"                                       When pinning young objects, the objects are clonned into the old space.\n"
+"                                       When pinning young objects, the objects are cloned into the old space. (default: false)\n"
 "                                       It tries to allocate the object in a segment with already pinned objects.\n"
-"	                                    Avoid the clonning process avoid this search and allocate the clonned object anywhere?\n"
+"	                                  Avoid the clonning process avoid this search and allocate the clonned object anywhere?\n"
```
Comment (guillep) [inline src/parameters/parameters.c:453]: Avoid double negation in the `avoidSearchingSegmentsWithPinnedObjects` option

**Pull request merged**